src/codecs/html/README.md

```diff
@@ -38,3 +38,126 @@ Some of the main external dependencies:
 - `js-beautify` for pretty generated HTML.
 
 - `json5` for more human readable representation of arbitrary values.
+
+---
+
+## Microdata and semantic tag usage (WIP)
+
+### Schema nodes to add microdata to
+* Article
+    * `div` `itemtype='schema:Article'`
+    * itemprops:
+        * authors* (author?): `ol`
+        * title*: `h1`? (+ `title`)
+        * environment: `data`?
+* Audio
+    * `audio`
+    * itemprops:
+        * caption*: `figcaption`
+        * transcript
+* Brand
+    * `picture`/`img` `itemtype='schema:Brand'`
+    * itemprops:
+        * name: maybe unnecessary, just use in alt tag?
+        * logo*: `source`/`image`
+        * reviews
+* Code
+    * `div` `itemtype='stencila:Code'`
+    * itemprops:
+        * language* (maybe important for setting syntax highlighting): `data`
+        * value*: `code`
+* CodeChunk
+    * `div` `itemtype='stencila:CodeChunk'`
```

Comment:
According to the spec the [recommended way](https://html.spec.whatwg.org/multipage/grouping-content.html#the-pre-element) for code blocks is to use a `<pre><code>...</code></pre>`

Also note the second example in this sections which shows the usage of the `<samp>` tag for outputs